spec/models/invoice_spec.rb

```diff
@@ -164,42 +164,15 @@
     end
   end
 
-  describe '#wallet_transaction_amount' do
-    let(:customer) { create(:customer) }
-    let(:invoice) { create(:invoice, customer:, organization: customer.organization) }
-    let(:wallet) { create(:wallet, customer:, balance: 10.0, credits_balance: 10.0) }
-    let(:wallet_transaction) do
-      create(:wallet_transaction, invoice:, wallet:, amount: 1, credit_amount: 1)
-    end
-
-    before { wallet_transaction }
-
-    it 'returns the wallet transaction amount' do
-      expect(invoice.wallet_transaction_amount.to_s).to eq('1.00')
-    end
-  end
-
   describe '#subtotal_before_prepaid_credits' do
     let(:customer) { create(:customer) }
-    let(:invoice) { create(:invoice, customer:, amount_cents: 555, organization: customer.organization) }
-    let(:wallet) { create(:wallet, customer:, balance: 10.0, credits_balance: 10.0) }
-    let(:wallet_transaction) do
-      create(:wallet_transaction, invoice:, wallet:, amount: 1, credit_amount: 1)
+    let(:invoice) do
+      create(:invoice, customer:, amount_cents: 555, organization: customer.organization, prepaid_credit_amount_cents: 100)
```

Comment:
Layout/LineLength: Line is too long. [123/120] (https://rubystyle.guide#max-line-length)